realsense2_camera/test/live_camera/test_camera_failing_tests.py

```diff
@@ -0,0 +1,268 @@
+# Copyright 2023 Intel Corporation. All Rights Reserved.
```

Comment:
in a second thought, I think this file should not be exists.
The best option is to open a sperate ticket in Jira for this, and to handle these formats/streams issues later.
on my machines, I could not make these configurations fail.

Reply:
removed